windows/hcsshim.go

```diff
@@ -147,53 +146,6 @@ func newWindowsContainerConfig(ctx context.Context, owner, id string, spec *spec
 	return conf, nil
 }
 
-// newLinuxConfig generates a hcsshim Linux container configuration from the
-// provided OCI Spec
-func newLinuxConfig(ctx context.Context, owner, id string, spec *specs.Spec) (*hcsshim.ContainerConfig, error) {
```

Comment:
I think this is a work in progress.

Reply:
cc @darrenstahlmsft

Reply:
Yes, this is used by https://github.com/containerd/containerd/pull/1455 which is on hold for now while I'm focusing on Windows container bringup.

Reply:
Why include unexported code before it is used? Can't it be re-added when it is actually needed?

Reply:
It was generated by a split of existing code into Windows and LCOW parts. The reason it was added in a PR that it was not consumed in was to create smaller PRs that are easier to read and faster to merge (The large LCOW PR that it would otherwise have been in is still not merged, as it is backed up on a number of other changes).

It can be removed and re-added later.